packages/api/endpoints/providers.js

```diff
@@ -87,26 +86,20 @@ async function post(req, res) {
 
   const id = apiProvider.id;
 
+  let postgresProvider;
   try {
     let record;
     if (!apiProvider.id) {
       throw new ValidationError('Provider records require an id');
     }
-
-    const postgresProvider = await translateApiProviderToPostgresProvider(apiProvider);
+    postgresProvider = await translateApiProviderToPostgresProvider(apiProvider);
     validateProviderHost(apiProvider.host);
 
-    try {
-      await createRejectableTransaction(knex, async (trx) => {
-        await providerPgModel.create(trx, postgresProvider);
-        record = await providerModel.create(apiProvider);
-        await indexProvider(esClient, record, process.env.ES_INDEX);
-      });
-    } catch (innerError) {
-      // Clean up DynamoDB record in case of any failure
-      await providerModel.delete(apiProvider);
-      throw innerError;
-    }
+    await createRejectableTransaction(knex, async (trx) => {
+      const [updatedPostgresProvider] = await providerPgModel.create(trx, postgresProvider);
```

Comment:
I think it's crypted in the pgmodel create.  I mean I know it is because I tested it and looked in the PG records to verify.  Let me see if I can find where.